Sources/GDSCommon/Patterns/GDSInformation/GDSInformationViewModel.swift

```diff
@@ -32,3 +32,8 @@ public protocol GDSInformationViewModel {
     var primaryButtonViewModel: ButtonViewModel { get }
     var secondaryButtonViewModel: ButtonViewModel? { get }
 }
+
+@MainActor
+public protocol GDSInformationViewModelWithChildView {
+    var childView: UIView? { get }
```

Comment:
Please can we make this non-optional?

One of the main points of having a separate protocol is to reduce our use of optionals; it becomes optional to conform to the protocol rather than the property itself being optional.